packages/@orchidui-vue/src/Form/Input/OCInput.vue

```diff
@@ -23,84 +22,97 @@ defineProps({
     type: String,
     default: "Label",
   },
+  modelValue: {
+    type: String,
+    default: "",
+  },
   inlineLabel: {
     type: Boolean,
     default: false,
   },
   hint: {
     type: String,
-    default: "This is a hint text to help user.",
+    default: "",
   },
   placeholder: {
     type: String,
     default: "Placeholder",
   },
   icon: {
     type: String,
-    default: "circle",
+    default: "",
   },
   type: {
     type: String,
-    default: "leading", // "trailing" | "leading"  | ""
+    default: "",
   },
 });
+defineEmits({
+  "update:modelValue": [],
+});
 </script>
 
 <template>
-  <div class="flex flex-col gap-y-1">
+  <div class="flex flex-col gap-y-2">
     <!--  Label  -->
-    <span v-if="!inlineLabel" class="text-xs text-oc-grey-400">{{
-      label
-    }}</span>
+    <span
+      v-if="!inlineLabel"
+      class="flex items-center h-[18px] font-medium text-oc-text-400"
+      >{{ label }}</span
+    >
 
     <!--  Input wrapper  -->
     <div
-      class="rounded-lg h-9 border flex items-center gap-x-2 px-2"
+      class="rounded-lg h-[36px] border flex items-center gap-x-3 px-3"
       :class="[
         {
-          'shadow-[0_0_0_2px] shadow-oc-grey-200': isFocused,
+          'shadow-[0_0_0_2px]': isFocused,
         },
-        !disabled && isError ? 'border-oc-error' : 'border-oc-grey-200',
-        disabled ? 'bg-oc-bg' : 'bg-white',
+        !disabled && isError
+          ? 'border-oc-error shadow-oc-error'
+          : 'border-oc-grey-200 shadow-oc-grey-200',
+        disabled ? 'bg-oc-bg-dark pointer-events-none' : 'bg-white',
       ]"
     >
       <slot v-if="type === 'trailing'" name="trailing">
         <Dropdown>
           <template #trigger>
             <MenuButton
-              class="py-2 pr-2 border-r border-oc-grey-200 text-xs font-medium flex items-center gap-x-1 text-oc-grey-400"
+              class="py-3 pr-3 border-r border-oc-grey-200 text-sm font-medium flex items-center gap-x-2 text-oc-text-400"
             >
-              USD
```

Comment:
we need currency as props, for input can we have base input (included label, hint, error message, and slot for input type)